components/VolunteersSection/index.jsx

```diff
@@ -0,0 +1,15 @@
+const { default: Volunteers } = require('../../pages/[lng]/volunteers');
```

Comment:
dont mix import styles if it is possible, please use `import` instead of `require` here